rules/windows/command_and_control_remote_file_copy_desktopimgdownldr.toml

```diff
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2020/09/03"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies desktopimgdownldr utility being used to download a remote file. Verify details such as parent process, url
+reputation, and downloaded file details.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Remote File Download via Desktopimgdownldr Utility"
+references = ["https://labs.sentinelone.com/living-off-windows-land-a-new-native-file-downldr/"]
+risk_score = 47
+rule_id = "15c0b7a7-9c34-4869-b25b-fa6518414899"
+severity = "medium"
+tags = ["Elastic", "Windows"]
+type = "query"
+
+query = '''
+event.category:process and event.type:(start or process_started) and
+ (process.name:desktopimgdownldr.exe or process.pe.original_file_name:desktopimgdownldr.exe) and process.args:/lockscreenurl\:http*
```

Comment:
```suggestion
 (process.name:desktopimgdownldr.exe or process.pe.original_file_name:desktopimgdownldr.exe) and process.args:\/lockscreenurl\:http*
```

Can you test it too to make sure it works as expected

Reply:
escaping / is not needed:
Not working:
![image](https://user-images.githubusercontent.com/64742097/92302976-1b80f300-ef71-11ea-9d8a-ac277da11086.png)
Without escaping works:
![image](https://user-images.githubusercontent.com/64742097/92302992-48cda100-ef71-11ea-946c-af1ed618b574.png)